docs/alias.connect.md

```diff
@@ -0,0 +1,11 @@
+It's common for browsers to try to prefix domain names with ```www```. Your site doesn't allow for this, so ```www.yoursite.com``` leads to a 404 error page.
```

Comment:
Would give a error if any status code other than 200. So could be not able to connect or a error code